docs/_config.yml

```diff
@@ -7,15 +7,15 @@ url: "https://shopify.github.io"
 sass:
   cache: true
 # remember to update the checksums
-themekitversion: "v1.0.2"
+themekitversion: "v1.1.0"
```

Comment:
@tanema Should this have been v1.0.3 already? The 'manual downloads' section on the site is still pointing to v1.0.2: https://shopify.github.io/themekit/

Reply:
yeah, it should have been but it is clear that I never got around to it.